src/components/Search/SearchAutocomplete.tsx

```diff
@@ -46,7 +75,41 @@ export default function SearchAutocomplete() {
               />
             </ComboboxButton>
           </div>
-          <ComboboxOptions className="ring-opacity/5 absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black focus:outline-none sm:text-sm">
+
+          {/* Source Dropdown Menu */}
+          {showDropdown && (
```

Comment:
we are using shadcn component library is there a reason you are not using that here? https://ui.shadcn.com/docs/components/dropdown-menu

Reply:
There was no identifiers in the code that we are using that besides this schema:
  "$schema": "https://ui.shadcn.com/schema.json",

Reply:
I don't understand. can you explain?

Reply:
sorry please check out https://ui.shadcn.com/ and then run `npx shadcn add ...`